Revit_Core_Engine/Query/ViewSolid.cs

```diff
@@ -35,15 +35,17 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [PreviousVersion("7.3", "BH.Revit.Engine.Core.Query.ViewSolid(Autodesk.Revit.DB.View)")]
         [Description("Returns a solid that represents the 3-dimensional extents of a given view.")]
         [Input("view", "View to compute the solids.")]
+        [Input("createIfUncropped", "Create solid if crop view property is inactive. New solid will be extended by maximum extends value in directions perpendicular to the view type.")]
         [Output("solid", "Solid that represents the 3-dimensional extents of the input view.")]
-        public static Solid ViewSolid(this View view)
+        public static Solid ViewSolid(this View view, bool createIfUncropped = false)
```

Comment:
Now looking at the method signature, I would consider being even more explicit in parameter naming - the more informative we are, the easier to use the method without digging in descriptions 👍 
```suggestion
        public static Solid ViewSolid(this View view, bool createUnlimitedIfViewUncropped = false)
```